.github/workflows/camunda-platform-release.yml

```diff
@@ -751,6 +751,65 @@ jobs:
       dockerImage: ${{ (inputs.dryRun || inputs.releaseProcessDryRun) && '' || format('camunda/zeebe:{0}', inputs.releaseVersion) }}
     secrets: inherit
 
+  # This job waits for the existing FOSSA scan (triggered by check-licenses.yml on tag push)
+  # then creates releases and generates attribution/SBOM reports for single-app only
+  # Flags impact:
+  # - With dryRun=true or releaseProcessDryRun=true: Runs in dry-run mode (no actual release creation)
+  # - Skipped for non-stable versions (containing '-' like SNAPSHOT, alpha, rc)
+  fossa-release:
+    name: Create FOSSA releases and generate attribution/SBOM reports
+    runs-on: ubuntu-latest
+    needs: [release, github, zeebe-docker, operate-docker, tasklist-docker, camunda-docker, snyk]
+    if: ${{ !contains(inputs.releaseVersion, '-') }}
```

Comment:
is this for excluding RCs from being covered here? RCs are public releases (e.g. https://hub.docker.com/layers/camunda/camunda/8.8.0-rc4/images/sha256-b133f2e0c2b6f0a32369a0ab175f82d272b5b82d64b380977e823b01195d1a44), I think no reason to skip it there.